### PR TITLE
bug fixing for admin v1

### PR DESCRIPTION
**Pull request by denisdimitrov**
Some of the urls were missing in the `venuePaths` array which was being used to get the `venueId` in the hook. Because of this, the `venueId` was `undefined` when it wasn't. Also the url of the venue creation has been changed because there was a collision between: `/admin/venue/:venueId` and `/admin/venue/creation` 

Fixes https://github.com/sparkletown/internal-sparkle-issues/issues/194

### CODE REVIEW

File: src/components/organisms/AppRouter/AppRouter.jsx
Comment (0xdevalias) [inline src/components/organisms/AppRouter/AppRouter.jsx:65]: Yaaaas! I've wanted to change this for SO long! 🎉

File: src/hooks/useVenueId.ts
Comment (0xdevalias) [inline src/hooks/useVenueId.ts:14]: [Potentially OOS] I backlogged https://github.com/sparkletown/internal-sparkle-issues/issues/164 in the past, about how we should be able to get rid of this 'paths' hack entirely.

Essentially, if we use `useParams`, it will be able to extract `:venueId` no matter where it exists in the url path:

- https://reactrouter.com/web/api/Hooks/useparams

Which I believe might even reduce this hook down to basically a single line.

File: src/pages/Admin/Venue/Rooms/RoomsForm.tsx
Comment (0xdevalias) [inline src/pages/Admin/Venue/Rooms/RoomsForm.tsx:62]: Remove these `console.log`s

File: src/pages/Admin/Venue/VenueWizard.tsx
Comment (0xdevalias) [inline src/pages/Admin/Venue/VenueWizard.tsx:58]:
```suggestion
```

File: src/pages/Admin/Venue/VenueWizard.tsx
Comment (0xdevalias) [inline src/pages/Admin/Venue/VenueWizard.tsx:105]:
```suggestion
```

**Review by 0xdevalias — CHANGES_REQUESTED** [review]
[CRO] A few little changes + suggestions to make this more robust

File: src/hooks/useVenueId.ts
Comment (denisdimitrov) [reply src/hooks/useVenueId.ts:14]: I agree with this but lets fix it in a separate PR.

**Comment by denisdimitrov** [comment]
@0xdevalias The console logs have been removed and the conversations about them have been resolved. I agree with the venueId param but lets do it in another PR.

**Review by mike-lvov — APPROVED** [review]

**Comment by denisdimitrov** [comment]
@0xdevalias I think merging is blocked because of your previous request changes. Please merge this as soon as you approve it because Thomas needs these changes.

File: src/hooks/useVenueId.ts
Comment (0xdevalias) [reply src/hooks/useVenueId.ts:14]: Sure

**Comment by 0xdevalias** [comment]
> @0xdevalias I think merging is blocked because of your previous request changes. Please merge this as soon as you approve it because Thomas needs these changes.

Yup, that does appear to be why merging is blocked at the moment. I'll review/approve/merge in a sec.

While in general it's good practice to let the original reviewer re-review to remove this in the 'normal way' (so that they can ensure all of their concerns were adequately fixed), in 'special cases' where it's an urgent bug fix/similar, it's fine to use GitHub's 'dismiss review' feature:

image

When you use this, please make sure to include a bit of context/description so that we can look back and see why that review was dismissed in future:

image

If a review was dismissed in this way, but the PR was still approved by another dev overall, then I don't consider this a 'break glass' fix, and so you don't need to apply that label (see below for more details)

---

At a high level, the 'engineering culture' I want to encourage is that each individual contributor is empowered to be able to make the changes they need, as they need to. When we have the luxury of time, then I definitely want to make sure we follow our standard review flows (which is approved by at least 1 other dev), but I also trust the judgement of the team to know when it is ok to 'step outside those flows' to unblock something. In the cases where we do 'step outside those flows', I want to make sure that we have enough context/traceability to be able to see where that happened, and why (eg. standardizing how we 'mark' an issue when we use a 'break glass' approval bypass (see https://github.com/sparkletown/internal-sparkle-issues/issues/13#issuecomment-739734800), etc)

(I'll also make sure to copy/reference this in https://github.com/sparkletown/sparkle/pull/1062 so we can document it properly going forward)

**Review by 0xdevalias — APPROVED** [review]
[CRO] Previous review comments were fixed up. LGTM!

File: src/hooks/useVenueId.ts
Comment (0xdevalias) [reply src/hooks/useVenueId.ts:14]: FYI: I believe https://github.com/sparkletown/internal-sparkle-issues/issues/194#issuecomment-765080430 is related to the change I requested here.

Comment (0xdevalias) [reply src/hooks/useVenueId.ts:14]: see https://github.com/sparkletown/sparkle/pull/1091 for fix

**Pull request merged**
